client/src/components/Donation/donation-modal.tsx

```diff
@@ -77,359 +62,69 @@ const Illustration = ({
   );
 };
 
-function getctaNumberBetween1To10() {
-  const min = 1;
-  const max = 10;
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-}
```

Comment:
The new CTA is performing better to the CTA rotation feature, so all of the related logic is removed.